tests/lax_numpy_vectorize_test.py

```diff
@@ -216,6 +217,13 @@ def test_inconsistent_output_size(self):
         ValueError, r"inconsistent size for core dimension 'n'"):
       f(jnp.zeros((2, 3)), jnp.zeros((3, 4)))
 
+  def test_expand_dims_multiple_outputs_no_signature(self):
+    f = jnp.vectorize(lambda x: (x, x))
+    x = jnp.arange(1)
+    xx = f(x)
+    self.assertAllClose(xx[0], x)
```

Comment:
Can we also do `self.assertIsInstance(xx, tuple)`?

Reply:
ok